addons/point_of_sale/models/pos_order.py

```diff
@@ -864,7 +864,7 @@ class PosOrderLineLot(models.Model):
     _description = "Specify product lot/serial number in pos order line"
     _rec_name = "lot_name"
 
-    pos_order_line_id = fields.Many2one('pos.order.line')
+    pos_order_line_id = fields.Many2one('pos.order.line', auto_join=True)
```

Comment:
@nle-odoo 
since we recently discussed `auto_join=True`, could you review this PR please? Thank you